From 569f75127aa0c18236a07a1151016e2c122cfd94 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Thu, 2 Mar 2023 13:47:15 +0200
Subject: [PATCH 44/54] Revert "fdts: s32cc: use dt-bindings defines for PHI1
 and PER clks"

This reverts commit 9135e2f7872c7108ded912539b57b022c125a55b.

The ADC support in ATF is deleted as it's no longer needed for
identifying the RDB revision to apply a DDR fixup.

Issue: ALB-9722
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 4 ++--
 fdts/s32g3.dtsi | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index e88e3fb2b..cc69d8b43 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -412,7 +412,7 @@
 				<0>,
 				<0>,
 				<0>,
-				<S32GEN1_PER_FREQ>,
+				<80000000>,
 				<40000000>,
 				<40000000>,
 				<80000000>,
@@ -466,7 +466,7 @@
 			assigned-clock-rates =
 				<0>,
 				<S32GEN1_PERIPH_PLL_VCO_FREQ>, <100000000>,
-				<S32GEN1_PERIPH_PLL_PHI1_FREQ>, <80000000>,
+				<80000000>, <80000000>,
 				<125000000>, <200000000>,
 				<125000000>, <100000000>;
 		};
diff --git a/fdts/s32g3.dtsi b/fdts/s32g3.dtsi
index f014937a6..50ef11773 100644
--- a/fdts/s32g3.dtsi
+++ b/fdts/s32g3.dtsi
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause)
 /*
- * Copyright 2021-2022 NXP
+ * Copyright 2021-2023 NXP
  */
 #include <dt-bindings/clock/s32g3-clock.h>
 
@@ -176,7 +176,7 @@
 		<0>,
 		<0>,
 		<0>,
-		<S32GEN1_PER_FREQ>,
+		<80000000>,
 		<0>,
 		<40000000>,
 		<40000000>,
-- 
2.25.1

